app/scripts/main.js

```diff
@@ -0,0 +1,73 @@
+jQuery(function($) {
+  //Initiat WOW JS
+  new WOW().init();
+  // one page navigation 
+  $('.main-navigation').onePageNav({
+    currentClass: 'active'
+  });
+  // Countdown
+  $('#counter').bind('inview', function(event, visible, visiblePartX, visiblePartY) {
```

Comment:
Line is too long.<br>'visiblePartY' is defined but never used.<br>'visiblePartX' is defined but never used.